src/player.js

```diff
@@ -130,6 +146,30 @@ export default class Player extends FakeEventTarget {
    */
   _el: HTMLDivElement;
   /**
+   * The player text DOM element container.
+   * @type {HTMLDivElement}
+   * @private
+   */
+  _textDisplayEl: HTMLDivElement;
```

Comment:
should clean on destroy ? or can rely vid element destroy?

Reply:
No need, we clean the content on reset and this is just the placeholder